triagebot.toml

```diff
@@ -181,6 +181,11 @@ trigger_files = [
     "x.py",
     "src/bootstrap",
     "src/tools/rust-installer",
+    "configure",
+    "Cargo.toml",
+    "Cargo.lock",
```

Comment:
Is this one supposed to be here? Adding any dependency to any rustc crate now makes the bot add the `A-bootstrap` label.

Reply:
https://github.com/rust-lang/rust/pull/106958